### DISCUSSION TITLE
save mp4 with vedio and audio

### DISCUSSION THREAD

**Discussion by lekeeith** [bluenviron/gortsplib#488] [Questions]
in the example, it is easy for me to download mp4audio or mp4vedio from rtsp. but how to save full mp4 with audio and vedio ?

**Comment by aler9** [comment]
Hello, it's enough to merge the MPEG-TS muxers of the two examples you mentioned together:

https://github.com/bluenviron/gortsplib/blob/main/examples/client-play-format-h264-save-to-disk/mpegts_muxer.go
https://github.com/bluenviron/gortsplib/blob/main/examples/client-play-format-mpeg4audio-save-to-disk/mpegts_muxer.go

The examples are not intended for a ready-to-use solution, but they can be easily adapted into one.
